src/components/cards/sidebar/monthlyBadgesCard.tsx

```
@@ -12,53 +72,47 @@ export default function MonthlyBadgesCard() {
+                        "October",
+                        "November",
+                        "December",
+                    ];
+
+                    const monthlyQuestDate = new Date(monthlyQuest.startDate);
+                    const monthlyQuestMonthIndex = monthlyQuestDate.getMonth();
+                    const monthlyQuestMonth = months[monthlyQuestMonthIndex];
+                    const monthlyQuestYear = monthlyQuestDate.getFullYear();
+
+                    return (
+                        <div className="monthly-badge">
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)